inspirehep/modules/workflows/tasks/merging.py

```diff
@@ -95,3 +95,14 @@ def merge_articles(obj, eng):
         obj.extra_data['callback_url'] = \
             get_resolve_merge_conflicts_callback_url()
     obj.save()
+
+
+def conflicts_ticket_context(user, obj):
+    server_name = current_app.config['SERVER_NAME']
+    workflow_id = obj.id
+    subject = "Merge conflict on workflow {} needs to be resolved.".format(workflow_id)
```

Comment:
could you reuse the logic from the core curation tickets which display arXiv ID, DOI, etc.?

Reply:
I would prefer not to reuse whole context from there, but I can update this one with more data from the workflow to display arXiv, DOI , ID etc.

Reply:
Is this solution better? @michamos